libshaderc_spvc/src/spvcir_pass.cc

```diff
@@ -14,25 +14,189 @@
 
 #include "spvcir_pass.h"
 
-#include <fstream>
+#include "source/opt/ir_context.h"
 
 namespace spvtools {
 namespace opt {
 
+SpvcIrPass::SpvcIrPass(spirv_cross::ParsedIR &ir) {
```

Comment:
Google style guide says that reference parameters should be const.
Since you'll be modifying this ir object, change the interface to make it pointer-to-spirv_cross::ParsedIR
That way it's clear that you'll be modifying it.